test/java/seedu/addressbook/testutil/TestMain.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.testutil;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import seedu.addressbook.commands.*;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.parser.Parser;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Test class for Main
+ */
+public class TestMain {
```

Comment:
This PR is for testing the `FindCommand` class, not the '`find` command' as typed by the user. Hence, I don't see the need for this class. Compare with #118